src/gulpfile.js

```diff
@@ -29,6 +29,18 @@ gulp.task('build:zip', (done) => {
 });
 
 /**
+ * Does a rebuild of your theme's locale files and runs translation
+ * tests on each file using @shopify/theme-lint
+ *
+ * @function test
+ * @memberof slate-cli.tasks
+ * @static
+ */
+gulp.task('test', (done) => {
+  runSequence('build:locales', 'lint:locales', done);
```

Comment:
Two questions:

1. Would it be possible to just test on the `src` directory since it looks like build just copies files over? If you think of JS or SCSS, you'd want to lint the `src` dir.
2. Does `build:locales` need to also be included on the build task now? Seems weird that it would only be required here.